server/src/config/database.ts

```
@@ -0,0 +1,206 @@
+
+  // Redis Connection
+  public async connectRedis(): Promise<Redis> {
+    try {
+      if (this.redisClient) {
+        return this.redisClient;
+      }
+
+      const redisUrl = process.env.REDIS_URL || 'redis://localhost:6379';
+      
+      this.redisClient = new Redis(redisUrl, {
+        maxRetriesPerRequest: 3,
```

Biome (1.9.4)

[error] This property value named maxRetriesPerRequest is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named maxRetriesPerRequest


(lint/suspicious/noDuplicateObjectKeys)